web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -1066,25 +1066,6 @@ describe("profile - starting business", () => {
     ).not.toBeInTheDocument();
   });
 
-  it("prevents user from saving when petcare is selected as industry, but essential question is not answered", async () => {
```

Comment:
[Dust] shouldn't we keep this test? the essential question is still required right?

Reply:
It's being tested on line 1109.